### PR TITLE
Propose to create SIG-AI in KubeEdge community

### PR DESCRIPTION
**Pull request by jaypume**
1. Add meeting infomation for sig ai.
2. Add charter for sig ai.

### CODE REVIEW

**Comment by ugvddm** [comment]
/assign @kevin-wangzefeng
/cc @gklp 
please review it

File: sig-ai/charter.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+SIG AI is responsible to provide general platform capabilities based on KubeEdge so  that AI applications running at the edge can benefit from cost reduction, model performance improvement, and data privacy protection.
+
+### In scope
+
+#### Areas of Focus
+
+- work with existing AI framework, to enable it run better on KubeEdge, such as Tensorflow, PaddlePaddle, onnx etc.
+    - KubeEdge supports heterogeneous hardware, such as Ascend, Kunglun, Cambrian, and Rockchip.
+    - KubeEdge supports ONNX models, such as tutorials.
+    - Cooperation with other open source communities, such as Akraino
```
Comment (kevin-wangzefeng) [inline sig-ai/charter.md:16]:
```suggestion
- Collaborate with AI framework open source communities, to help better running on KubeEdge, e.g. Tensorflow, Pytorch, PaddlePaddle, onnx etc. Works including but not limited to:
    - support heterogeneous hardware, such as Ascend, Kunglun, Cambrian, and Rockchip.
    - support ONNX models, such as tutorials.
```

File: sig-ai/charter.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+SIG AI is responsible to provide general platform capabilities based on KubeEdge so  that AI applications running at the edge can benefit from cost reduction, model performance improvement, and data privacy protection.
+
+### In scope
+
+#### Areas of Focus
+
+- Collaborate with AI framework open source communities, to help better running on KubeEdge, e.g. Tensorflow, Pytorch, PaddlePaddle, onnx etc. Works including but not limited to:
+    - support heterogeneous hardware, such as Ascend, Kunglun, Cambrian, and Rockchip.
+    - support ONNX models, such as tutorials.
+- provide an **edge-cloud collaborative** AI framework based on KubeEdge capabilities, such as model and dataset.
```
Comment (fisherxu) [inline sig-ai/charter.md:16]:
```suggestion
- Maintaining **edge-cloud collaborative** AI APIs and tools based on KubeEdge capabilities, such as model and dataset.
```

File: sig-ai/charter.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+- provide an end-to-end edge-cloud collaborative AI benchmarking framework, for typical AI applications scenarios on KubeEdge.
```
Comment (fisherxu) [inline sig-ai/charter.md:17]:
```suggestion
- Maintaining end-to-end edge-cloud collaborative AI benchmarking, for typical AI applications scenarios on KubeEdge.
```

File: sig-ai/README.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
+# SIG AI
+
+SIG AI is responsible to provide general platform capabilities based on KubeEdge so  that AI applications running at the edge can benefit from cost reduction, model performance improvement, and data privacy protection.
```
Comment (fisherxu) [inline sig-ai/README.md:3]: Also update the intro here

File: sig-ai/charter.md [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+    - Integrate KubeFlow and ONNX into KubeEdge, to enable interoperability of edge models with diverse formats
+    - Cooperate with other open source communities, e.g., Akraino and LF AI
+1. Research **includes but not limited to**:
+    - Cloud training and edge inference
+    - Incremental learning
+    - Edge-cloud-collaborative inference
+        - Knowledge distillation for the cloud and edge model
+    - Federated learning 
+    - Edge model and dataset management
+1. Edge AI benchmarking relevant work, to identify most important dimensions when developing, evaluating Edge AI applications-and-services system:
```
Comment (kevin-wangzefeng) [inline sig-ai/charter.md:23]:
```suggestion
1. Edge AI benchmarking relevant work, to help users identify most important dimensions when developing, evaluating Edge AI applications-and-services system:
```

File: sig-ai/charter.md [hunk 6]
```diff
@@ -0,0 +1,45 @@
...
+    - Integrate typical AI frameworks into KubeEdge, e.g., Tensorflow, Pytorch, PaddlePaddle and Mindspore etc.
+    - Integrate KubeFlow and ONNX into KubeEdge, to enable interoperability of edge models with diverse formats
+    - Cooperate with other open source communities, e.g., Akraino and LF AI
+1. Research **includes but not limited to**:
+    - Cloud training and edge inference
+    - Incremental learning
+    - Edge-cloud-collaborative inference
+        - Knowledge distillation for the cloud and edge model
+    - Federated learning 
+    - Edge model and dataset management
```
Comment (kevin-wangzefeng) [inline sig-ai/charter.md:22]:
```suggestion
1. Synergy models research for AI workloads, **including but not limited to**:
    - Cloud training and edge inference
    - Edge-cloud-collaborative inference
        - Knowledge distillation for the cloud and edge model
    - Incremental learning
    - Federated learning
    - Edge model and dataset management
```

File: sig-ai/README.md (hunk 4 above)
Comment (kevin-wangzefeng) [inline sig-ai/README.md:3]:
```suggestion
The SIG AI focuses on technical discussion, API definition, reference architecture, implementation in the Edge AI field, to enable AI applications better running on edge (including cost saving, performance improvement, and data protection).
```

File: sig-ai/charter.md [hunk 7]
```diff
@@ -0,0 +1,45 @@
...
+The SIG AI focuses on technical discussion, API definition, reference architecture, implementation in the Edge AI field, to enable AI applications better running on edge (including cost saving, performance improvement, and data protection).
+    
+### In scope
+#### Areas of Focus
+1. Empower KubeEdge with existing AI ecosystems, to support execution of Edge AI applications and services：
+    - Support heterogeneous edge hardware, e.g., Ascend, Kunlun, Cambrian, and Rockchip
+    - Integrate typical AI frameworks into KubeEdge, e.g., Tensorflow, Pytorch, PaddlePaddle and Mindspore etc.
+    - Integrate KubeFlow and ONNX into KubeEdge, to enable interoperability of edge models with diverse formats
+    - Cooperate with other open source communities, e.g., Akraino and LF AI
+1. Synergy models research for AI workloads, **including but not limited to**:
```
Comment (kevin-wangzefeng) [inline sig-ai/charter.md:16]:
```suggestion
1. Synergy mechanisms research for AI workloads, **including but not limited to**:
```

**Pull request merged**